x-pack/plugins/task_manager/server/task_pool.ts

```diff
@@ -148,7 +149,11 @@ export class TaskPool {
   private cancelExpiredTasks() {
     for (const task of this.running) {
       if (task.isExpired) {
-        this.logger.debug(`Cancelling expired task ${task.toString()}.`);
+        this.logger.warn(
+          `Cancelling task ${task.toString()} as it expired at ${task.expiration.toISOString()}${
+            task.startedAt ? ` after running for ${moment(task.startedAt).toNow(true)}` : ``
```

Comment:
nit: If we can make the message more precise instead of logging `after running for a few seconds`.

Reply:
Yeah, I was trying to decide whether moment's humanize is more pleasant than printing out the duration in a `mm:ss` format. Could definitely do the latter.